batch/scripts/create_vector_schema.sh

```diff
@@ -8,11 +8,20 @@ set -e
 # -s | --source
 # -l | --source_layer
 # -f | --local_file
+# -m | --field_map
 # -X | --zipped
 
+# optional arguments
+# -g | --geometry_name (get_arguments.sh specifies default)
+# -i | --fid_name (get_arguments.sh specifies default)
+
 ME=$(basename "$0")
 . get_arguments.sh "$@"
 
+set -u
+
+# FIXME: Downloading the whole file for this step might be unnecessary
```

Comment:
I think you would need to unless it's CSV? You would need to the whole shapefile or geodatabase downloaded to access its schema.

Reply:
Good news! ogrinfo seems to be able to get the schema from some remote file types! I think this change could be back on the table (for the future).

Reply:
Nice! Did you happen to create a Jira card for this `FIXME`? 😉

Reply:
I have now! https://gfw.atlassian.net/browse/GTC-2233